playbooks/rename-master-to-main.md

```diff
@@ -129,3 +129,9 @@ project's "stages":
 
 - Rename the `master` stage to `main`
 - Input `main` as the branch name as well
+
+### External tools
+
+Since we haver external tools that are not possible to map by checking the repository code itself, this is something we need to consider as a Team duties after rename as well.
```

Comment:
```suggestion
Since we have external tools that are not possible to map by checking the repository code itself, this is something we need to consider as a Team duties after rename as well.
```